cowtown2023/src/main/java/com/team2357/frc2023/subsystems/ShooterSubsystem.java

```diff
@@ -3,17 +3,42 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.team2357.frc2023.Constants;
+import com.team2357.frc2023.Constants.SHOOTER;
+import com.team2357.lib.subsystems.LimelightSubsystem;
+import com.team2357.lib.util.RobotMath;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ShooterSubsystem extends SubsystemBase {
 
     private CANSparkMax m_topShooterMotor;
     private CANSparkMax m_bottomShooterMotor;
+    private SHOOTER m_config;
+    private boolean m_isClosedLoopEnabled;
```

Comment:
Make ShooterSubsystem extend ClosedLoopSubsystem. Then we don't have to make our own variables and functions to do the same thing.